packages/ocamlfind/ocamlfind.1.3.2/opam

```diff
@@ -20,5 +24,5 @@ depends: [
   "conf-m4" {build}
   "num" {= "0"}
 ]
-available: ocaml-version > "3.12.2"
+available: [ocaml-version > "3.12.2" & ocaml-version < "4.07"]
```

Comment:
The opam file of 1.3.3 says
``` 
available: [(ocaml-version >= "3.08") & (ocaml-version <= "4.01.0")] # ocamlfind uses Arg.align of 3.08
```
I think 1.3.2 should be at least as restricted as 1.3.3

Reply:
You're probably right. Done